app/interactors/check_answer.rb

```diff
@@ -0,0 +1,18 @@
+class CheckAnswer
+  include Interactor
+
+  def call
+    card = Card.find_by_id(context.id)
```

Comment:
Устаревший синтаксис. Следует использовать `find_by(id: ..)` Странно, что рубокоп на это не ругнулся тебе.
Ну вот представь, что `card` не нашлась, такой карточки нету в базе — вот это некорректное поведение, мы дальше не можем продолжить работать, и вот это как раз таки повод сделать context.fail!

```ruby
card = Card.find_by(id: context.id)
return context.fail! if card.nil?

if compare_answers(card)
  card.update(review_date: 3.days.since(Time.now))
  context.notice = t('.correct')
else
  context.notice = t('.wrong')
end
```